Sources/Testing/Test+Macro.swift

```diff
@@ -484,6 +484,27 @@ extension Test {
   }
 }
 
+// MARK: - Test pragmas
+
+/// A macro used similarly to `#pragma` in C or `@_semantics` in the standard
+/// library.
+///
+/// - Parameters:
+///   - arguments: Zero or more context-specific arguments.
+///
+/// The use cases for this macro are subject to change over time as the needs of
+/// the testing library change. The implementation of this macro in the
+/// TestingMacros target determines how different arguments are handled.
+///
+/// - Note: This macro has compile-time effects _only_ and should not affect a
+///   compiled test target.
+///
+/// - Warning: This macro is used to implement the `@Test` macro. Do not use it
+///   directly.
```

Comment:
Strictly speaking, isn't it used to implement by `#expect` macros, not `@Test`? Maybe this should be more vague

Reply:
Yes, no, maybe, both :)

Right now it's actually used to implement one of our tests only.

Reply:
```suggestion
/// - Warning: This macro is used to implement other macros declared by the testing
///   library. Do not use it directly.
```